<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>647</width>
    <height>241</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTableWidget" name="_arrWindow">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>50</y>
      <width>441</width>
      <height>141</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="_insertSortBtn">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>80</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>вставка</string>
    </property>
   </widget>
   <widget class="QPushButton" name="_bubbleSortBtn">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>20</y>
      <width>80</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>бульбашка</string>
    </property>
   </widget>
   <widget class="QPushButton" name="_clrBtn">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>140</y>
      <width>80</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>очистити</string>
    </property>
   </widget>
   <widget class="QPushButton" name="_undo_btn">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>200</y>
      <width>80</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>крок назад</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>30</y>
      <width>161</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>введіть к-ть елементів</string>
    </property>
   </widget>
   <widget class="QPushButton" name="_sortNoSteps_btn">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>200</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>пуск</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>180</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>пуск без &quot;кроків&quot;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="_goButt">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>80</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>зібрати </string>
    </property>
   </widget>
   <widget class="QPushButton" name="_mergeSortBtn">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>20</y>
      <width>80</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>злиття </string>
    </property>
   </widget>
   <widget class="QPushButton" name="_rndBtn">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>110</y>
      <width>80</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>рандом</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>151</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>выберите сортировку</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="_elemCount">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>50</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="_redo_btn">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>200</y>
      <width>80</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>крок вперед</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
